src/common/tags.js

```diff
@@ -48,12 +48,16 @@ export default class Tags extends Component {
                 autoFocus
                 style={{maxWidth: '4em', margin: '2px'}}
                 onKeyDown={event => {
-                  const { target } = event
+                  const { keyCode, target } = event
 
-                  if (event.keyCode === 13 && target.value) {
+                  if (keyCode === 13 || keyCode === 27) {
+                    event.preventDefault()
```

Comment:
It makes sense but I would prefer if we would test twice for both 13 and 27, is there a way to better write that?